ios/RNZoomUs.m

```diff
@@ -257,10 +278,103 @@ - (void)onClickShareScreen:(UIViewController *)parentVC {
 
 - (BOOL)respondsToSelector:(SEL)aSelector
 {
-    if (aSelector == @selector(onClickShareScreen:)) {
-        return screenShareExtension != nil;
-    }
-    return [super respondsToSelector:aSelector];
+  if (aSelector == @selector(onClickShareScreen:)) {
+    return screenShareExtension != nil;
+  }
+  return [super respondsToSelector:aSelector];
+}
+
+#pragma mark - React Native event emitters and event handling
+
+- (void)startObserving {
+  hasObservers = YES;
+}
+
+- (void)stopObserving {
+  hasObservers = NO;
+}
+
+- (NSArray<NSString *> *)supportedEvents {
+  return @[@"AuthEvent", @"MeetingEvent"];
+}
+
+- (void)sendEventWithName:(NSString *)name event:(NSString *)event {
+  if (hasObservers) {
+    [self sendEventWithName:name body:@{@"event": event}];
+  }
+}
+
+- (NSString *)authErrorName:(MobileRTCAuthError)error {
+  switch (error) {
+    case MobileRTCAuthError_Success: return @"success";
+    case MobileRTCAuthError_KeyOrSecretEmpty: return @"keyOrSecretEmpty"; // iOS only
+    case MobileRTCAuthError_KeyOrSecretWrong: return @"keyOrSecretWrong"; // iOS only
+    case MobileRTCAuthError_AccountNotSupport: return @"accountNotSupport"; // iOS only
+    case MobileRTCAuthError_AccountNotEnableSDK: return @"accountNotEnableSDK"; // iOS only
+    case MobileRTCAuthError_ServiceBusy: return @"serviceBusy"; // iOS only
+    case MobileRTCAuthError_None: return @"none"; // iOS only
+    case MobileRTCAuthError_OverTime: return @"overTime"; // iOS only
+    case MobileRTCAuthError_NetworkIssue: return @"networkIssue"; // iOS only
+    case MobileRTCAuthError_ClientIncompatible: return @"clientIncompatible";
```

Comment:
in order to make the mappings to be more readable. put common ios/android cases on top and sort all of the return alphabetically.
In this way it will be easier to compare what is in java and what is in objective-c

Reply:
Updated.